server/src/reaction/error.rs

```diff
@@ -0,0 +1,18 @@
+#[derive(Debug, thiserror::Error)]
+pub enum Error {
+    #[error("Not found")]
+    NotFound,
+    #[error("Database error")]
+    Sqlx(#[from] sqlx::Error),
+}
+
+impl From<Error> for tonic::Status {
+    fn from(value: Error) -> Self {
+        match value {
+            Error::NotFound => tonic::Status::not_found("Not found"),
+            Error::Sqlx(_) => tonic::Status::internal("Database error"),
```

Comment:
#58 と同じ感じで`tracing::error!`入れてください :pray:

```suggestion
            Error::Sqlx(e) => {
                tracing::error!(error = &e as &dyn std::error::Error);
                tonic::Status::internal("Database error")
            }
```